xpart/particles/particles.py

```diff
@@ -344,7 +344,7 @@ def classical_particle_radius(self):
         Method to calculate classical particle radius from particles
         """
         m0 = self.mass0*1.782661921e-36  # electron volt - kg conversion
```

Comment:
Perhaps we should use qe/c**2 to calculate the eV - kg conversion instead of hardcoding it.

Reply:
Good idea, fixed it now!